### PR TITLE
qweb3 0.2.6 src to dist changes

### PR DESCRIPTION
**Pull request by dwalintukan**

### CODE REVIEW

File: src/contracts/bodhi_token.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import _ from 'lodash';
-import Contract from 'qweb3/src/contract';
+import Contract from 'qweb3/dist/contract';
```
Comment (chrisli30) [inline src/contracts/bodhi_token.js]: No, actually we need to update qweb3 to module.exports contract in index.js. That way this line's syntax becomes "import {Contract} from 'qweb3'" and need to change file path any more.

Comment (dwalintukan) [reply src/contracts/bodhi_token.js]: nice, done! I noob at JS

File: index.js [hunk 2]
```diff
@@ -3,7 +3,7 @@ import moment from 'moment';
 import promise from 'bluebird';
 import restify from 'restify';
 import corsMiddleware from 'restify-cors-middleware';
-import Qweb3 from 'qweb3';
+import { Qweb3 } from 'qweb3';
```
Comment (chrisli30) [inline index.js]: The same as bodhi-graphql comments. Qweb3 import should be 
import Qweb3 from 'qweb3';

Comment (dwalintukan) [reply index.js]: fixed

**Pull request merged**